libs/blocks/gnav/gnav.js

```diff
@@ -448,6 +449,28 @@ class Gnav {
     }
   };
 
+  setBreadcrumbSEO = () => {
+    const breadcrumb = this.el.querySelector('.breadcrumbs');
+    if (breadcrumb) {
+      const isSeo = breadcrumb.classList.contains('seo');
```

Comment:
Was there author discussion around this? I'd think that breadcrumbs would be an always generating JSON-LD type thing.

Reply:
@fullcolorcoder Thx for reviewing. I addressed the opt-out option based on @auniverseaway comment below with https://github.com/adobecom/milo/commit/8c56a2ef09464fe0ec5e617e113115b7d71c4476.